reactor-core/src/test/java/reactor/core/publisher/OnDiscardShouldNotLeakTest.java

```diff
@@ -45,24 +45,50 @@
 @RunWith(Parameterized.class)
 public class OnDiscardShouldNotLeakTest {
 
-	//add DiscardScenarios here to test more operators
-	private static DiscardScenario[] SCENARIOS = new DiscardScenario[] {
-			DiscardScenario.allFluxSourceArray("merge", 4, Flux::merge),
-			DiscardScenario.fluxSource("onBackpressureBuffer", 1, Flux::onBackpressureBuffer),
-			DiscardScenario.rawSource("flatMapInner", 1, raw -> Flux.just(1).flatMap(f -> raw)),
-			DiscardScenario.fluxSource("flatMap", 1, main -> main.flatMap(f -> Mono.just(f).hide().flux())),
-			DiscardScenario.fluxSource("flatMapIterable", 1, f -> f.flatMapIterable(Arrays::asList)),
-			DiscardScenario.fluxSource("publishOnDelayErrors", 1, f -> f.publishOn(Schedulers.immediate())),
-			DiscardScenario.fluxSource("publishOnImmediateErrors", 1, f -> f.publishOn(Schedulers.immediate(), false, Queues.SMALL_BUFFER_SIZE)),
-			DiscardScenario.fluxSource("unicastProcessor", 1, f -> f.subscribeWith(UnicastProcessor.create()))
-	};
-
-	private static boolean[][] CONDITIONAL_AND_FUSED = new boolean[][] {
-			{ false, false },
-			{ true, false },
-			{ false, true },
-			{ true, true }
-	};
+    //add DiscardScenarios here to test more operators
```

Comment:
formatting: there shouldn't be spaces in this file if you use the official Reactor intellij format

Reply:
🙃